web/styles/components.css

```diff
@@ -6,7 +6,7 @@
         padding: 0;
         display: inline-block;
         position: relative;
-        vertical-align: top;
+        vertical-align: middle;
```

Comment:
Have you verified all the places where we use `checbox` inside `new-style` class? The ones in settings look fine but I have not checked others.

Reply:
Did the frontend conversation determine we wanted to do this only for the one checkbox?

Reply:
Okay I misinterpreted the thing. I have applied the css for this specific checkbox now.